tests/testthat/_snaps/compare-data-frame.md

```diff
@@ -0,0 +1,70 @@
+# informative diff for additions and deletions
+
+    Code
+      df <- data.frame(x = 1:5, y = 5:1)
+      compare(df, unrowname(df[1:3, ]))
+    Output
+      `attr(old, 'row.names')`: 1 2 3 4 5
+      `attr(new, 'row.names')`: 1 2 3    
+      
+      old vs new
+              x y
+        [1, ] 1 5
+        [2, ] 2 4
+        [3, ] 3 3
+      - [4, ] 4 2
+      - [5, ] 5 1
+      
+      `old$x`: 1 2 3 4 5
+      `new$x`: 1 2 3    
+      
+      `old$y`: 5 4 3 2 1
+      `new$y`: 5 4 3    
+    Code
+      compare(df, unrowname(df[c(1, 5, 2, 3, 4, 5), ]))
+    Output
+      `attr(old, 'row.names')[3:5]`: 3 4 5  
+      `attr(new, 'row.names')[3:6]`: 3 4 5 6
+      
+      old vs new
+              x y
+        [1, ] 1 5
+      + [2, ] 5 1
```

Comment:
Similar to the previous example, it's not super obvious that `new` has an insertion.

Reply:
And this:

```
             x z
  old[1, ]   1 a
- old[2, ]   2 b
+ new[2, ] 100 B
  old[3, ]   3 c
```

Reply:
This helps (the explicit use of `old` and `new`.